fava_investor/modules/performance/balances.py

```diff
@@ -8,8 +8,8 @@ def get_closed_tree_with_value_accounts_only(accapi, config):
     filter_tree(tree, accounts_to_keep)
     return tree
 
-def get_value_accounts_and_parents(tree, accounts, pattern ):
-    result = set([acc for acc in accounts if re.match(pattern, acc)])
+def get_value_accounts_and_parents(tree, accounts, pattern):
+    result = set([acc for acc in accounts if any([re.match(pattern, acc)])])
```

Comment:
Remove any([])

Reply:
thats silly, I have pay more attention  :smiley:

Reply:
haha, no worries, it's easy to miss these.

Merged, thanks! BTW, feel free to amend previous commits and force push the change to open PRs, especially when they are super-small changes. I do this so that most commits have meaningful changesets.